receipt_agent/receipt_agent/agents/label_evaluator/llm_review.py

```
@@ -694,28 +693,21 @@ def review_issues_batch(
-        if rate_limiter:
-            response = rate_limiter.invoke([HumanMessage(content=prompt)])
-        else:
-            response = llm.invoke([HumanMessage(content=prompt)])
+    # Call LLM with structured output
+    from receipt_agent.prompts.structured_outputs import BatchedReviewResponse
 
-        response_text = response.content.strip()
-        return parse_batched_llm_response(
-            response_text, len(normalized_issues)
-        )
+    caller = rate_limiter if rate_limiter else llm
```

Ruff (0.15.0)

[error] Local variable `caller` is assigned to but never used

Remove assignment to unused variable `caller`

(F841)